src/main.rs

```diff
@@ -202,6 +201,9 @@ fn run_app<B: Backend>(
                 event => break event,
             }
         };
+
+        start_time = Utc::now().time();
```

Comment:
Probably it's better to use [Instant](https://doc.rust-lang.org/stable/std/time/struct.Instant.html). Like

```rust
let t = Instant::now();
// work
println!("elapsed: {:?}", t.elapsed());
```